core/main/src/Core/Container.ts

```diff
@@ -393,4 +396,9 @@ export class Container {
 
         this.density = (canvas.width * canvas.height) / (densityOptions.factor * pxRatio * densityOptions.area);
     }
+
+    private loadTheme(): void {
```

Comment:
This method needs a `name?: string` property, when passed searches the theme in the options with that name, when `undefined` (not passed) searches the `default` theme in the options